deploy/playbooks/roles/repos/templates/distributions

```diff
@@ -0,0 +1,87 @@
+Codename: sid
+Suite: stable
+Components: main
+Architectures: amd64 armhf i386 source
+Origin: Inktank
+Description: Ceph distributed file system
+DebIndices: Packages Release . .gz .bz2
+DscIndices: Sources Release .gz .bz2
+Contents: .gz .bz2
+SignWith: 17ED316D
+
+Codename: wheezy
+Suite: stable
+Components: main
+Architectures: amd64 armhf i386 source
+Origin: Inktank
+Description: Ceph distributed file system
+DebIndices: Packages Release . .gz .bz2
+DscIndices: Sources Release .gz .bz2
+Contents: .gz .bz2
+SignWith: 17ED316D
+
+Codename: squeeze
+Suite: stable
+Components: main
+Architectures: amd64 armhf i386 source
+Origin: Inktank
+Description: Ceph distributed file system
+DebIndices: Packages Release . .gz .bz2
+DscIndices: Sources Release .gz .bz2
+Contents: .gz .bz2
+SignWith: 17ED316D
+
+Codename: raring
+Suite: stable
+Components: main
+Architectures: amd64 armhf i386 source
+Origin: Inktank
+Description: Ceph distributed file system
+DebIndices: Packages Release . .gz .bz2
+DscIndices: Sources Release .gz .bz2
+Contents: .gz .bz2
+SignWith: 17ED316D
+
+Codename: quantal
+Suite: stable
+Components: main
+Architectures: amd64 armhf i386 source
+Origin: Inktank
+Description: Ceph distributed file system
```

Comment:
So, one var in defaults to hold this value. Maybe, ``distributions_description``?

Would we ever need a distributions file that describes multiple projects or is that even possible?